### DISCUSSION TITLE
What about the not accurate version number of the games on Skyline ?

### DISCUSSION THREAD

**Discussion by Wirufurido** [skyline-emu/skyline#2060] [Q&A]
Hello everyone !

I was just wondering about the version number of the games which is not accurate.

For exemple, I have cuphead 1.3.4 + the DLC that I compiled using bx64. Yuzu on my computer shows 1.3.4 + the DLC but Skyline show me 1.2.0. Is it just a visual bug or my game is downgraded to 1.2.0 ?

Thanks for you answer !

**Comment by Mononono87** [comment]
Skyline does not support adding dlc right now

**Reply by Wirufurido** [reply]
Okay, so does that mean that I should not use compiled XCI (Base + DLC) ? 

What about the update, it is not considered as a DLC, so for games without DLCs, it should show the right version number.
